src/penn_chime/presentation.py

```diff
@@ -433,9 +433,15 @@ def subscribe(st_obj):
     affiliation = st_obj.text_input (label="Enter Affiliation", value="", key="na_upper_2")
     if st_obj.button (label="Submit", key="ta_submit_1"):
         row = [email, name, affiliation]
-        send_subscription_to_google_sheet(st_obj, row)
+        send_subscription_to_google_sheet_secret_json(st_obj, row)
 
-def send_subscription_to_google_sheet(st_obj, row):
+def send_subscription_to_google_sheet_secret_json(st_obj, row):
+    json_secret = "/Users/mpatil763/opensource/COVED-19/repo/chime/src/penn_chime/client_secret.json"
```

Comment:
This is pointed at the wrong directory. Should be:

```suggestion
    json_secret = "/mnt/google-api-creds/client_secret.json"
```

Reply:
This has been updated.

Reply:
Reviewed the changes in st_display.py.